torch/csrc/distributed/rpc/types.h

```diff
@@ -0,0 +1,42 @@
+#pragma once
+
+#include <ATen/core/ivalue.h>
+#include <atomic>
+
+namespace torch {
+namespace distributed {
+namespace rpc {
+
+using worker_id_t = int16_t;
+using local_id_t = uint64_t;
+
+struct GloballyUniqueId final {
```

Comment:
Can we use `operator int()` to implicitly convert this to int64_t? This way we don't need to worry about Hash etc. and can easily use this object as an int.